Plugin/CppApi/source/ArcGISArView.cpp

```diff
@@ -180,10 +182,11 @@ Point ArcGISArView::screenToLocation(const QPoint& screenPoint) const
 }
 
 /*!
-  \brief Register the QML creatable types provide by QR toolkit.
+  \brief Register the QML-creatable types provided by QR toolkit.
```

Comment:
I didn't see that previously, it should be `AR` not `QR`.

Reply:
This comment still applies.